src/core/lib/config/core_configuration.h

```diff
@@ -21,7 +21,9 @@
 
 #include "src/core/lib/channel/channel_args_preconditioning.h"
 #include "src/core/lib/channel/handshaker_registry.h"
+#include "src/core/lib/gprpp/sync.h"
```

Comment:
I think this include isn't needed anymore.

Reply:
done